File: MORM.Java/src/MORM/Java/Models/Venda/Caixamov.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "CAIXAMOV")
public class Caixamov extends CollectionItem
{
    @CampoAttribute(Campo = "ID_CAIXA", Tipo = CampoTipo.Key)
    public Integer Id_Caixa;
    @CampoAttribute(Campo = "NR_SEQ", Tipo = CampoTipo.Key)
    public Integer Nr_Seq;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "TP_LANCTO", Tipo = CampoTipo.Req)
    public Integer Tp_Lancto;
    @CampoAttribute(Campo = "VL_LANCTO", Tipo = CampoTipo.Req)
    public Float Vl_Lancto;
    @CampoAttribute(Campo = "NR_DOC", Tipo = CampoTipo.Req)
    public Integer Nr_Doc;
    @CampoAttribute(Campo = "DS_AUX", Tipo = CampoTipo.Req)
    public String Ds_Aux;
}

File: MORM.CSharp/MORM.CSharp/Models/Venda/Caixamov.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("CAIXAMOV")]
    public class Caixamov : CollectionItem
    {
        [Campo("ID_CAIXA", CampoTipo.Key)]
        public int Id_Caixa { get; set; }
        [Campo("NR_SEQ", CampoTipo.Key)]
        public int Nr_Seq { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("TP_LANCTO", CampoTipo.Req)]
        public int Tp_Lancto { get; set; }
        [Campo("VL_LANCTO", CampoTipo.Req)]
        public double Vl_Lancto { get; set; }
        [Campo("NR_DOC", CampoTipo.Req)]
        public int Nr_Doc { get; set; }
        [Campo("DS_AUX", CampoTipo.Req)]
        public string Ds_Aux { get; set; }
    }
}